build.bash

```diff
@@ -240,16 +245,17 @@ if [ "$hw_platform" == "pi-raspios32" ] || [ "$hw_platform" == "pi-raspios64beta
   fi
   check_command_availability_and_exit "$REQ_COMMANDS" "$REQ_PACKAGES"
 
-  if [ -f "$zipfile" ]; then
-    echo_process "Using local copy of Raspberry Pi OS ($bits-bit) Lite image, $zipfile... "
-    cp $zipfile "$buildfolder/$zipfile"
+  if [[ $(basename "$(curl "$baseurl" -s -L -I  -o /dev/null -w '%{url_effective}')") == "$(ls -- *-raspios-*.zip 2> /dev/null)" ]]; then
```

Comment:
build process is still based on wget, maybe we can stick to wget or move to curl? (I know wget is a bit more tricky to read the effective url, see my current PR)

Reply:
I can probably do that, lets stick with wget for now.

Reply:
Actually, lets move to cURL.